### PR TITLE
Provide a more step-by-step instruction for getting started

### PR DESCRIPTION
**Pull request by lenhard-thomas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ also need to build a docker image.
 
 First you need to get Gerrit to generate an API cookie; you can do this by:
 
-- Going to your Gerrit's webpage
-- Clicking on "Generate Password"
+- Going to your Gerrit's dashboard
```
Comment (dlatypov) [inline README.md]: this is very much a nit, but I'd prefer the imperative tense now we have more steps.
i.e.
```
First you need to get Gerrit to generate an API cookie. To get one:
- Go to your Gerrit instance's dashboard
- Click on "Settings"
...
```

Comment (dlatypov) [inline README.md]: nit: perhaps include an example like "(e.g. https://linux-review.googlesource.com/)" or maybe directly link to https://linux-review.googlesource.com/settings ?

**Review by dlatypov — COMMENTED** [review]
As noted by the failing check, we'll need to figure out the CLA situation before you can submit any code.

Also, some minor nits/suggestions

File: README.md (hunk 1 above)
Comment (lenhard-thomas) [reply README.md]: Definitely agree! Updated the steps to be in imperative tense.

Comment (lenhard-thomas) [reply README.md]: Added a direct link.

**Review by dlatypov — APPROVED** [review]
Looks good, thanks!

I forgot the code relied on a "./gerritcookies" here, https://github.com/google/lkml-gerrit-bridge/blob/bb4052b300736fbcf87d220b1ff83b559b9702e1/src/main.py#L36
So the dockerfile change also makes sense to me.

**Comment by dlatypov** [comment]
Actually, I forgot the look at the commit description.
Since this is a single commit, we'll probably just "rebase and merge", so it would be nice if the commit description could explain the dockerfile change.

It can be something short like 1bde2f5e05be601ca33ec3697df6a298f426d204.
Note: you don't need the "Signed-off-by" line like that has. That's just force of habit on Brendan's part from LKML.

**Review by bjh83 — APPROVED** [review]

**Pull request merged**